qiskit-superstaq/qiskit_superstaq/superstaq_provider.py

```diff
@@ -483,3 +483,7 @@ def supercheq(
     def target_info(self, target: str) -> Dict[str, Any]:
         """Returns information about device specified by `target`."""
         return self._client.target_info(target)["target_info"]
+
+    def get_targets(self) -> Dict[str, Any]:
+        """Gets list of available targets."""
```

Comment:
nit: 
```suggestion
        """Gets a list of available targets."""
```

Reply:
I originally had that but wanted to keep it in parity with css service's endpoint where the docstring is "Gets list of available targets." since there's only 1 list.

Reply:
Oh I see, that makes sense